### PR TITLE
Show environment usage

### PR DESCRIPTION
**Pull request by Timple**
and show `default` can also have a source

### CODE REVIEW

**Review by MatthijsBurgh — DISMISSED** [review]
Though I am not sure the comment about ROS2 is true. We didn't test that yet.

File: README.md [hunk 1]
```diff
@@ -121,7 +140,7 @@ Taking the above into account, the following combinations for `install.yaml` are
        source:
          type: system
          name: <Package name>
-     indigo:
+     noetic:
```
Comment (Timple) [inline README.md:143]: Why did you flip it back from the default?

Now the example is not clear that `default` can also have something other than `pass`.

Comment (MatthijsBurgh) [reply README.md:143]: That was not clear from your changes.

**Review by MatthijsBurgh — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Timple) [reply README.md]: I tried to catch that in the description:
> and show default can also have a source.

But failed I guess :smile:

Comment (MatthijsBurgh) [reply README.md]: Haha, 'read the manual' == 'read the commit message'

Comment (Timple) [reply README.md]: Well descriptive commit messages are important!

**Pull request merged**